alphadia/data/bruker.py

```diff
@@ -65,13 +65,20 @@ def __init__(
                     mmap_detector_events,
                 )
 
-                if self._cycle.shape[0] != 1:
+                try:
+                    cycle_shape = self._cycle.shape[0]
+                except AttributeError as e:
                     logger.error(
-                        "Unexpected cycle shape. Will only retain first frame group"
-                    )
-                    raise ValueError(
-                        "Unexpected cycle shape. Will only retain first frame group"
+                        "Could not find cycle shape. Please check if this is a valid DIA data set."
                     )
+                    raise e
+                else:
+                    if cycle_shape != 1:
+                        msg = (
+                            "Unexpected cycle shape. Will only retain first frame group"
```

Comment:
@GeorgWa The error message sounds like processing will continue, but apparently it does not .. shall we rephrase it?

Reply:
Yes, let's rephrase this to make it clear that we fail here.